src/logger.ts

```diff
@@ -0,0 +1,34 @@
+export interface Logger {
```

Comment:
Docs.

Reply:
This is all internal but adding docs for our sake.